tests/Errors.js

```diff
@@ -3,37 +3,56 @@
 /*
  * This module loads Error and FailureInfo enum from ErrorReporter.sol.
  */
-
+const fs = require('fs')
 const path = require('path');
-const solparse = require('solparse');
+const solparse = require('@solidity-parser/parser');
 
 const errorReporterPath = path.join(__dirname, '..', 'contracts', 'ErrorReporter.sol');
-const contents = solparse.parseFile(errorReporterPath);
+const errorReporterContest = fs.readFileSync(errorReporterPath)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
const errorReporterContest = fs.readFileSync(errorReporterPath);
```